cnames_active.js

```diff
@@ -2875,6 +2875,7 @@ var cnames_active = {
   "short": "shortit.netlify.app",
   "shortener": "jsmiith.github.io/shortener",
   "shortio": "ichiidev.github.io/short.io-docs",
+  "shoto": "cname.vercel-dns.com",
```

Comment:
```suggestion
  "shoto": "cname.vercel-dns.com", // noCF
```
As the CI error notes, you'll want the Cloudflare proxy disabled when using Vercel